src/remark/controllers/inputs/keyboard.js

```diff
@@ -79,25 +80,39 @@ Keyboard.prototype.addKeyboardEventListeners = function () {
         events.emit('gotoPreviousSlide');
         break;
       case 'b':
-        events.emit('toggleBlackout');
+        if (!self._disablePresentationShortcuts) {

```

Comment:
I think this would be better refactored to pass the events through a proxy function, that checks if _disabledPresentationShortcuts is enabled before emitting the event.

Also; personally; I'd rewrite the switch/case statement to use an object map of registered keys - and effectively make a lookup table - would reduce nesting, and avoid having to add another layer of nesting for the guard condition.